src/app/shared/podcast-nav/podcast-nav-dropdown.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { PodcastModel } from '../../ngrx';
+
+@Component({
+  selector: 'metrics-podcast-dropdown',
+  template: `
+    <div *ngIf="podcasts?.length > 1" class="dropdown" [class.open]="open">
+      <div class="overlay" (click)="toggleOpen()"></div>
+      <div class="dropdown-button">
+        <button (click)="toggleOpen()" >{{ selectedPodcast?.title }}<span class="down-arrow"></span></button>
+      </div>
+      <div class="dropdown-content rollout">
```

Comment:
Can we put some sort of max-height on this?  Noticing it gets pretty long when I'm logged in as admin.  (Granted, this isn't the normal use-case ... but if the dropdown-height > page-height, you can't get to the bottom items.)

Reply:
These rollout dropdowns with transition have a max height of 500px. https://github.com/PRX/metrics.prx.org/pull/164/files#diff-54a5c8e229b3bb725603f6b52bcff45dR51 I could make that shorter...? I took the transition off the popup one and set max-height to initial because I didn't want to constrain it if I didn't have to.

Reply:
Ah, okay, missed that (was testing with window < 500px tall).  As long as it has one, I'm good!